packages/plot/src/interactors/Highlight.js

```diff
@@ -80,6 +84,10 @@ export class Highlight {
       }
     }
   }
+
+  activate() {
+    return this.update();
```

Comment:
Why do we call update here?

Reply:
Good catch, that was a remnant I meant to remove. Originally I just had the activate function be a NOOP however thinking about it now since we have the `isActivatble` check we can also just remove the function all together since it doesn't actually make sense on the Highlight interactor. What do you think would be best?

Reply:
Just remove it then!